signature/src/keypair.rs

```diff
@@ -0,0 +1,46 @@
+pub trait KeyPair: core::fmt::Debug + Sized {
+    fn get_public_key_enc(&self) -> String;
```

Comment:
I think we should just name it `get_encoded_public_key`. Since `enc` could mean both encrypted or encoded in this context